docs/_tutorials/migration_v2.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Migrating to V2
+order: 1
+slug: migration-v2
+lang: en
+layout: tutorial
+permalink: /tutorial/migration-v2
+---
+# Migrating to v2.x
+
+<div class="section-content">
+This guide will walk you through the process of updating your app from using `bolt@v1.x` to `bolt@2.x`. 
```

Comment:
```suggestion
This guide will walk you through the process of updating your app from using `@slack/bolt@v1.x` to `@slack/bolt@2.x`. 
```

Reply:
Maybe a sentence like "There are a few changes you'll need to make but for most apps, these changes can be applied in 5 - 15 minutes."